app/controllers/group_purchases_controller.rb

```diff
@@ -0,0 +1,2 @@
+class GroupPurchasesController < ApplicationController
+end
```

Comment:
- [ ] Please delete any unused files or code for a cleaner and more optimized codebase. 👍